package.json

```diff
@@ -17,6 +17,9 @@
     "postmerge": "yarn install",
     "precommit": "git status --porcelain | awk 'match($1, \"[ACM]\"){print $2}' | grep '[js|jsx]$' | xargs eslint",
     "storybook": "start-storybook -p 9001 -c .storybook",
+    "storybook-static": "build-storybook -c .storybook -o .out",
```

Comment:
is `build-storybook` part of our existing dependencies, or do we need to add https://github.com/storybooks/storybook-deployer as a dev dependency?